src/test/ui/debuginfo/debuginfo_with_uninhabitable_field_and_unsized.rs

```diff
@@ -0,0 +1,28 @@
+// check-pass
+// compile-flags: -Cdebuginfo=2
+
```

Comment:
Please add a comment that this fixes https://github.com/rust-lang/rust/issues/94149.